File: backend/src/main/java/com/example/skillsim/SkillSimApplication.java
// 스프링 부트 애플리케이션을 구동하는 진입점 클래스
package com.example.skillsim;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class SkillSimApplication {

    public static void main(String[] args) {
        SpringApplication.run(SkillSimApplication.class, args);
    }
}

File: backend/src/main/kotlin/com/example/skillsim/SkillSimApplication.kt
// 스프링 부트 애플리케이션을 구동하는 진입점 클래스
package com.example.skillsim

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class SkillSimApplication

fun main(args: Array<String>) {
    runApplication<SkillSimApplication>(*args)
}
